### ISSUE TITLE
[Bug]: Color contrast for button selected state

### ISSUE THREAD

**Issue by Sarah4VT** [commercetools/nimbus#1580]
### Describe the bug

📌 Not sure if this is an _actual_ accessibility bug as I'm basing this off my own perception.

When displaying the "thumbs up" icon in a selected and unselected state, the background on the button doesn't have a lot of color contrast so it's hard for me to easily tell which one is selected.

### Nimbus Version Number

3.0.0

### Browser + OS

Mac, Chrome

### Priority

Medium

### To Reproduce

1. Project Settings
2. Channels
3. Open new chat
4. Send a message to agent, wait to get response
5. Feedback buttons appear for agent response

### Screenshots

<img width="1610" height="756" alt="Image" src="https://github.com/user-attachments/assets/4d41f644-6f43-4bc2-85e0-38fbf0a9c0de" />

### Additional context

You can see the agent here: https://mc.eu-central-1.aws.escemo.com/lifestyle-01/settings/project/channels
